Tests/test_file_mic.py

```diff
@@ -1,10 +1,46 @@
-from helper import unittest, PillowTestCase
+from helper import unittest, PillowTestCase, hopper
 
-from PIL import MicImagePlugin
+from PIL import Image, MicImagePlugin
+
+TEST_FILE = "Tests/images/hopper.mic"
 
 
 class TestFileMic(PillowTestCase):
 
+    def test_sanity(self):
+        im = Image.open(TEST_FILE)
+        im.load()
+        self.assertEqual(im.mode, "RGBA")
+        self.assertEqual(im.size, (128, 128))
+        self.assertEqual(im.format, "MIC")
+
+        im2 = hopper("RGBA")
+        self.assert_image_similar(im, im2, 123.5)
```

Comment:
I think we should go with this in the test, and note somewhere in the docs that there may be an embedded gamma of 2.2